web/src/utils/router.ts

```diff
@@ -43,13 +44,28 @@ const reporterToFlowdir = _.invert(flowdirToReporter);
 
 export const buildQueryArguments = (
   filters: Filter[],
+  forcedFilters: Filter[] | null = null,
   range: number | TimeRange,
   opts: QueryOptions
 ): QueryArguments => {
   // Note: at the moment the browser query params and API query params are tied together;
   // we may want to decouple them in the future.
   const params: QueryArguments = {};
-  _.each(filters, (f: Filter) => {
+  let allFilters: Filter[];
+  if (forcedFilters) {
```

Comment:
is there cases where we have `forcedFilters` AND some extra `filters` ? I thought if not, we could simplify and, from the caller, pass either `filters` or `forcedFilters`, but not both...

Reply:
`FlowDirection` and `Limit` are editable filters from the tabs now; so yes you can have both !

However you are not supposed to have forced & normal filters on the same columns but the user may try to edit URL arguments right ?

Reply:
But `FlowDirection` and `Limit` are not passed through `filters`, but through `opts`. They should not affect the search filters

Reply:
You're right, I will simplify that then :+1: 

Thanks for the feedback